python/pyspark/sql/readwriter.py

```diff
@@ -733,7 +733,7 @@ def save(self, path=None, format=None, mode=None, partitionBy=None, **options):
         :param partitionBy: names of partitioning columns
         :param options: all other string options
 
-        >>> df.write.mode('append').parquet(os.path.join(tempfile.mkdtemp(), 'data'))
+        >>> df.select("name", "favorite_color").write.save("namesAndFavColors.parquet")
```

Comment:
This is a code change which should not be mixed into here.
Please revert this change.

Reply:
I think it's a fix - this is doc for `.save` and this example doesn't match

Reply:
Oh. Got it. Thanks!